RELEASE.md

```diff
@@ -1,30 +1,34 @@
 # Release Instructions/Checklist
 
-This file describes the required steps to release neow3j.
-Before releasing make sure that the external documentation at [neow3j.io](https://neow3j.io) is up 
-to date.
+This file describes the required steps to release neow3j. Before releasing make sure that the external documentation
+at [neow3j.io](https://neow3j.io) is up-to-date.
 
 ## Release the Source Code
 
-- Bump the version on the develop branch (e.g., `develop-3.x`). Do a global search with the current 
-  version number and replace it with the new version number. There should be four affected 
-  locations in the `README.md`, one in the `build.gradle`, and one in `Compiler.java`.
-  - The tests NNSIntegrationTest, NFTIntegrationTest, DivisibleNFTIntegrationTest, 
-    and NonDivisibleNFTIntegrationTest will fail because of the version change in the compiler.
-   
-- Create a Pull Request from develop to master (e.g., from `develop-3.x` to `master-3.x`) -- this
-is called a "Release Pull Request"
-  - Set the name as "Release x.x.x"
-  - Set the correct milestone, project and a reviewer
-   
-- Review and merge the Release Pull Request, mainly checking for critical changes **or**
-changes that shouldn't be in the release
- 
-- Tag the main branch at the commit to be release (e.g., `3.2.1`)
- 
-- Right after merging, bump the version in the `build.gradle` file (`develop-3.x` branch)
+- Create a branch `release` from `main`.
```

Comment:
Maybe, `release-<VERSION>`? or `release` is enough since this branch will be deleted after? Not strongly opinionated on this one.

Reply:
`release` should be sufficient, since it's only used in a small timeframe anyways and then deleted.